src/components/DropdownColumn.jsx

```diff
@@ -1,17 +1,23 @@
 import React, { useState } from 'react'
 import {
-    Menu, MenuItem, Button, ListItemIcon, ListItemText, Grid,
+    Menu, MenuItem, Button, ListItemIcon, ListItemText, Grid, Snackbar
 } from '@material-ui/core'
+import Alert from '@material-ui/lab/Alert'
 import MoreHorizIcon from '@material-ui/icons/MoreHoriz'
 import Delete from '@material-ui/icons/Delete'
 import { useMutation, useApolloClient } from '@apollo/client'
 import { DELETE_COLUMN } from '../graphql/column/columnQueries'
 import { COLUMNORDER } from '../graphql/fragments'
+import { boardPageStyles } from '../styles/styles'
 
 const DropdownColumn = ({ columnId, boardId }) => {
     const [deleteColumn] = useMutation(DELETE_COLUMN)
     const [anchorEl, setAnchorEl] = useState(null)
     const client = useApolloClient()
+    const classes = boardPageStyles()
+    const [open, setOpen] = useState(false)
+    const snackbarMsg = `This action will permanently remove the selected column from your project and can't be later examined! Are you sure you want to delete it?`
```

Comment:
Tässä vois olla viestiä myös siitä, että sarakkeen sisällä olevat taskit hävii myös. Ja niille pitäis varmaan tehdä tietokannan tasolla kans jotain. Tuleeks deletoidun kolumnin kaikkiin taskeihin se soft delete deletedAt leima?